sos/report/__init__.py

```diff
@@ -506,6 +507,26 @@ def _get_block_devs(self):
             self.soslog.error("Could not get block device list: %s" % err)
             return []
 
+    def _get_tape_devs(self):
+        """Enumerate a list of tape devices on this system so that plugins
+        can iterate over them
+
+        These devices are used by add_device_cmd() in the Plugin class.
+        """
+        try:
+            devs = []
+            devdirs = [
+                'scsi_tape',
+                'lin_tape'
+            ]
+            for devdir in devdirs:
+                if os.path.isdir(f"/sys/class/{devdir}"):
```

Comment:
This is redundant test:

    >>> os.path.isdir("/blah")
    False
    >>> glob.glob("/blah")
    []
    >>> 

(purely from performance point of view, I even did a test comparing both approaches. It seems Python spends bit more time in `isdir` test every time than in running extra `glob` and adding empty list sometimes)